examples/wms-time.js

```diff
@@ -46,10 +45,10 @@ function updateInfo() {
 
 function setTime() {
   startDate.setMinutes(startDate.getMinutes() + 15);
-  if (startDate > Date.now()) {
+  if (+startDate > Date.now()) {
```

Comment:
```suggestion
  if (startDate > new Date()) {
```
Or, even better, assign something like
```js
const now = new Date();
```
outside the function and use it in the above suggestion instead of `new Date()`.

Reply:
This function advances the time in 15 minute steps and it is meant to keep following the current time. If you don't like `+startDate` then `startDate.getTime()` would also work.

Reply:
`startDate.getTime()` sounds great, thanks!